### DISCUSSION TITLE
Server detecting weird player activity

### DISCUSSION THREAD

**Discussion by opensourcefatguy** [SmartlyDressedGames/Unturned-3.x-Community#2364] [General]
Hey!

I've recently started hosting an unturned 3.x server, and so far there has been suspicious activity from certain players, i.e. having massive amounts of loot, despite the server only being up for about 4 days, finding/destroying any base that gets built

The telltale sign of my suspicion came from looking at the RCON console through my server hosting service
![fast](https://user-images.githubusercontent.com/78162814/106164202-bb752e00-6157-11eb-861b-481a74f06fd2.PNG)

I'm rather new to server hosting, so making heads or tail of this is challenging, all I can decipher is a player is moving/displacing quickly

Could this be a result of latency issues/memory of the server (most connections indicate these players have 20-50 ping average)
or is this a genuine exploit that they are using

Weirdly enough, most of these console lines are when me and my friends (who have admin status) are offline

Cheats are disabled, I cannot find anything online about this, just speculations

**Comment by SDGNelson** [comment]
By default the game has server-authoritative movement, so unless you disabled that cheaters cannot speed run / teleport. That warning (player moved ###) is from Rocket mod, and I see it occasionally on my personal test server when Rocket is enabled so I would not be concerned - it should probably be removed.

I do not think you need to worry, but if you do find that they are abusing something in the vanilla game (i.e. not a mod) then I will be happy to fix it.

**Reply by opensourcefatguy** [reply]
Alright, thanks! I saw console spitting out the same lines for a few people I knew werent cheating so I figured it was something else.
